Demo/Demo/Helpers/OrderRequestHelpers.swift

```diff
@@ -1,104 +1,97 @@
 import Foundation
-import PayPalCheckout
 
 enum OrderRequestHelpers {
 
-    static var orderAmount = 100.0
-
-    static func getOrderRequest(_ shippingPreference: OrderApplicationContext.ShippingPreference) -> OrderRequest {
-        return OrderRequest(
-            intent: .authorize,
+    static let orderAmount = 100.0
```

Comment:
nit: I don't think the PPCP sample merchant server is set up to auto void like BT yet. Do we want to set the order amount to like $1 incase we get a prod account stood up so we don't accidentally change ourselves $100 the first time we test? 🙈

Reply:
The PayPalNative & PayPalWeb demos aren't setup for Auth/Capture, I left a TODO for it.

But definitely agree once we get a prod account we should do that!